front/app/containers/MainHeader/Components/DesktopNavItems/DesktopNavbar.test.tsx

```diff
@@ -38,7 +38,7 @@ describe('<DesktopNavbar />', () => {
 
   it('renders correct number of navbar items', () => {
     render(<DesktopNavbar />);
-    expect(screen.getAllByTestId('desktop-navbar-item')).toHaveLength(5);
+    expect(screen.getAllByTestId('desktop-navbar-item')).toHaveLength(6);
```

Comment:
I reverted the change from this PR https://github.com/CitizenLabDotCo/citizenlab/pull/9039/files to keep the pipeline green, but not sure if this was the right thing to do.